checker/values.ml

```diff
@@ -383,7 +383,7 @@ and v_libobjt = Sum("Libobject.t",0,
      [| v_substobjs |];
      [| v_aobjs |];
      [| v_libobjs |];
-     [| List v_mp |];
+     [| Any |];
```

Comment:
Why does the checker care about libobject?

Reply:
cc @ppedrot

Reply:
The checker doesn't, but it's useful for `votour` to know about the internals for better debugging.